File: web3rpc/rpc-specs/code-samples/python/personal/deriveAccount.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

url = "url"
path = "path"
pin = True

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.derive_account(url, path, pin)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/deriveAccount.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const url = "url";
    const path = "path";
    const pin = true;

    provider.personal.deriveAccount(url, path, {pin}, (err, data, response) => {
        console.log(data);
    });
}
)()

